.env

```diff
@@ -1,4 +1,4 @@
 PYTHONUNBUFFERED=TRUE
-FLASK_APP=psi.wsgi:application
+FLASK_APP=psi.cli:application
```

Comment:
The `psi/wsgi.py` file should only be used for the Gunicorn/WSGI HTTP server. All else should go through the CLI.

This shouldn't break your Heroku deploy since Gunicorn is invoked separately via `Procfile` command

Reply:
Sounds reasonable here to decouple and separate wsgi and cli usage here.